pkgs/data/fonts/sahel-fonts/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "sahel-fonts";
```

Comment:
The `name` attribute usually includes the version also. Use `pname` here instead of `name`.